sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_analysis_client.py

```diff
@@ -83,9 +83,9 @@ def begin_analyze_document(self, model, document, **kwargs):
             "prebuilt-document", "prebuilt-layout".
         :param document: JPEG, PNG, PDF, TIFF, or BMP type file stream or bytes.
         :type document: bytes or IO[bytes]
-        :keyword list[str] pages: Custom page numbers for multi-page documents(PDF/TIFF). Input the page numbers
+        :keyword str pages: Custom page numbers for multi-page documents(PDF/TIFF). Input the page numbers
```

Comment:
latest swagger makes the change from list[str] to str. Since we're mostly just exposing the generated code here, we can do that change and discuss design later in beta2